app/models/game_night.rb

```diff
@@ -1,4 +1,10 @@
 class GameNight < ActiveRecord::Base
   validates :time, presence: true
   validates :location_name, presence: true
+
+  belongs_to :group
+  belongs_to :organizer, class_name: :User, foreign_key: :organizer_id
+
+  has_many :user_game_night_attendees
+  has_many :users, through: :user_game_night_attendees
```

Comment:
@Enriikke you'd mentioned the possibility of making these more generic (like `Memberships` in the case of Groups). What would that look like? Here, our `UserGameNightAttendees` has a `user_id` and a `game_night_id`. If you wanted something more versatile would you just have attributes like `id_1` and `id_2` so that a `Membership` or `Attendance` could apply to any type of model?